.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
_hee hee_ - but really (even though you might just be being silly), this is an endless debate that we could pursue, and this team has countless repos that already use spaces that we'd have to convert. I'm not opposed to tabs, but that this remains a war waged among incredibly prominent software gurus means we'll probably not be making any radical changes here. Additionally, CB as a whole seems to have settled on spaces over tabs.